open-ce/build_tree.py

```diff
@@ -93,6 +93,9 @@ def _create_recipes(repository, recipes, variant_config_files, variants, channel
     os.chdir(repository)
 
     config_data, _ = build_feedstock.load_package_config()
+    feedstock_conda_build_config_file = build_feedstock.get_conda_build_config()
+    if feedstock_conda_build_config_file:
+        variant_config_files.append(feedstock_conda_build_config_file)
```

Comment:
Okay, the recipe_config_file is being returned by `build_feedstock.load_package_config()` on line 95, but is being ignored.

I think this should be something more like:
```suggestion
    config_data, feedstock_conda_build_config_file  = build_feedstock.load_package_config()
    if feedstock_conda_build_config_file:
        variant_config_files.append(feedstock_conda_build_config_file)
```

But I'm still not sure if we should be modifying `variant_config_files`

Reply:
recipe_config_file returned by `build_feedstock.load_package_config()` is different than the one I want. I want `config/conda_build_config.yaml` whereas the config file returned by `load_package_config()` is `config/build-config.yaml`.

Reply:
This makes sense now. 
I don't think we should be modifying the input argument, as this will modify it in the calling function as well.

Reply:
I agree with @bethune-bryant , we shouldn't be modifying the input argument. We might not be using variant_config_files right now after calling _create_recipes, but if we do later on it might lead to some confusing behavior. We can just create a new local variable for this change to keep things cleaner.

Reply:
Okay. This is done.